docs/program-setup/mbed_targets.md

```diff
@@ -2,7 +2,7 @@
 
 Arm Mbed uses JSON as a description language for its build targets. You can find the JSON description of Mbed targets in `targets/targets.json` and in `custom_targets.json` in the root of a project directory. If you provide a source directory using the `--source` switch, Mbed looks for `custom_targets.json` in that directory instead. When you add new targets with `custom_targets.json`, they are added to the list of available targets, in addition to the [scanning rules](../program-setup/build-rules.html).
 
-<span class="notes">**Note:** The Online Compiler does not support this functionality. You need to use [Mbed CLI](../build-tools/mbed-cli-1.html) to take your code offline.</span>
+<span class="notes">**Note:** Keil Studio does not support this functionality. You need to use [Mbed CLI](../build-tools/mbed-cli-1.html) to take your code offline.</span>
```

Comment:
@federicobozzini @JoeTheGuitarist 
See this page: https://os.mbed.com/docs/mbed-os/v6.15/program-setup/adding-and-configuring-targets.html
Should we mention that Mbed Studio supports custom targets? 
Should we only mention Mbed CLI?

Reply:
Yes Mbed Studio supports this.